### PR TITLE
updated url for reporting invalid phishing site

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

Update to fix the Phish reporting list.

**Checklist**

* NA There is a related GitHub issue
* NA Tests are included if applicable
* NA Any added code is fully documented

**Screenshots/Recordings**
- MetaMask BlockList Url : poouoinm.tk
- PhishFort BlockList Url: aaaave.com

https://user-images.githubusercontent.com/10342624/172951561-4fb7346e-0a05-46fe-b1e7-658de3a5cd8c.mp4

### CODE REVIEW

File: app/components/Views/BrowserTab/index.js [hunk 1]
```diff
@@ -739,7 +745,9 @@ export const BrowserTab = (props) => {
    */
   const goToFilePhishingIssue = () => {
     setShowPhishingModal(false);
-    go(`https://github.com/metamask/eth-phishing-detect/issues/new`);
+    blockList.current === 'MetaMask' ? 
+    go(`https://github.com/metamask/eth-phishing-detect/issues/new`) : 
+    go(`https://github.com/phishfort/phishfort-lists/issues/new`);
```
Comment (gantunesr) [inline app/components/Views/BrowserTab/index.js:750]: Can we move this URLs to the constants file?

File: app/components/Views/BrowserTab/index.js [hunk 2]
```diff
@@ -411,9 +412,14 @@ export const BrowserTab = (props) => {
   const isAllowedUrl = useCallback(
     (hostname) => {
       const { PhishingController } = Engine.context;
+      const phishingControllerTestResult = PhishingController.test(hostname);
+
+      //Only assign the if the hostname is on the block list
```
Comment (gantunesr) [inline app/components/Views/BrowserTab/index.js:417]:
```suggestion
      // Only assign the if the hostname is on the block list
```

**Review by gantunesr — APPROVED** [review]
LGTM. Some minor nits

**Comment by Cal-L** [comment]
When you try to proceed to the website from the red phishing warning screen, you're prompted with the same warning screen a few seconds later. After the second attempt, you get the url.match warning.
imageimage

File: app/components/Views/BrowserTab/index.js (hunk 1 above)
Comment (sethkfman) [reply app/components/Views/BrowserTab/index.js:750]: Sure can good suggestion

**Comment by Cal-L** [comment]
Nice looks like the proceed forward is fixed. Something else seems to have broke though. I'm experiencing two strange scenarios:
1. Not sure if this is expected behavior but navigating to blacklisted website always shows the danger screen during the first app session. Once you kill the app and reopen however, navigating to these blacklisted screens no longer results in a danger screen showing (Of course this is after you proceed forward with these blacklisted sites in the first session).
2. In the first app session, navigating to a non-blacklisted site from a blacklisted site that you proceeded forward to wasn't working - as in there was no reaction. Navigating to another blacklisted site in this state did work though.

**Review by chrisleewilcox — APPROVED** [review]
LGTM

**Pull request merged**